spec/threema/receive/delivery_receipt_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Threema::Receive::DeliveryReceipt do
+  subject { described_class.new(**attributes) }
+  let(:attributes) { attributes_for(:delivery_receipt_receive) }
+
+  context '#content' do
+    it { is_expected.to respond_to(:content) }
+
+    it 'returns the content' do
+      expect(subject.content).to eq(attributes[:content])
+    end
+  end
+
+  context 'message ids' do
+    let(:attributes) { { content: "\x01\xE0\xC8\x12n8]\xF3\x19J\xB4+ \xC9\xAB|\xB6\xAAc#\v|Ro(".b } }
+
+    it 'returns an array of message ids' do
+      expect(subject.message_ids).to be_a(Array)
+    end
+
+    it 'with each message id' do
+      expect(subject.message_ids.length).to eq(3)
+    end
+
+    it 'with correct length' do
+      expect(subject.message_ids.all? { |string| string.length.eql?(16) }).to be(true)
+    end
```

Comment:
Make one test case out of it and show the explicit value. E.g.like:
```rb
      expect(subject.message_ids).to eq([1,2,3])
```

Reply:
I guess that makes sense. I feel like the value of the ids is not the important thing to get out of the test though.
I've added more info in the description.